src/disclosures/js/views/financial-view.js

```diff
@@ -241,11 +241,14 @@ var financialView = {
       $( '.content_graduate-program' ).hide();
       financialView.gradPlusVisible( false );
     }
-    this.jobPlacementVisible( values.jobRate !== '' );
-    if ( values.level.indexOf( 'Certificate' ) === 1 ) {
-      this.$degreeType.text( 'certificate' );
-    } else {
+    this.jobPlacementVisible(
+      values.jobRate !== 'None' && values.jobRate !== '' &&
+       typeof values.jobRate !== 'undefined'
```

Comment:
Trivial: remove extra space to align `values` and `typeof`. Also you might move the type check to be the first check that happens.